Following IPD4, man page sections have been changed from the old
Solaris/illumos scheme. Adjust sudo to suit.

diff -wpruN --no-dereference '--exclude=*.orig' a~/configure.ac a/configure.ac
--- a~/configure.ac	1970-01-01 00:00:00
+++ a/configure.ac	1970-01-01 00:00:00
@@ -1800,9 +1800,9 @@ case "$host" in
 		AS_IF([test "$with_AFS" = "yes"], [
 		    AFS_LIBS="-lc -lucb"
 		])
-		: ${mansectsu='1m'}
-		: ${mansectform='4'}
-		: ${mansectmisc='5'}
+		: ${mansectsu='8'}
+		: ${mansectform='5'}
+		: ${mansectmisc='7'}
 		test -z "$with_pam" && AUTH_EXCL_DEF="PAM"
 		AC_CHECK_FUNCS([priv_set], [PSMAN=1])
 		;;
